src/ui/popup.js

```diff
@@ -290,8 +290,8 @@ export default class Popup extends Evented {
         const offset = normalizeOffset(this.options.offset);
 
         if (!anchor) {
-            const width = this._container.offsetWidth;
-            const height = this._container.offsetHeight;
+            const width = this._container.clientWidth;
+            const height = this._container.clientHeight;
```

Comment:
This is measuring the popup's container rather than the map one, and we might want to leave `offset` sizes here, since for positioning the popup, it's important to take the whole popup into account (including borders/etc) rather than just its internal area.

Reply:
I reverted popups and the attribution and logo controls to use offsetWidth/Height